lib/chibi/test.scm

```diff
@@ -324,28 +315,30 @@
 
 ;;> \section{Accessors}
 
-;; (name (prop value) ...)
-(define (make-test-group name . o)
-  (let ((parent (and (pair? o) (car o)))
-        (group (list name (cons 'start-time (current-second)))))
-    (test-group-set! group 'parent parent)
-    (test-group-set! group 'verbose
-                     (if parent
-                         (test-group-ref parent 'verbose)
-                         (current-test-verbosity)))
-    (test-group-set! group 'level
-                     (if parent
-                         (+ 1 (test-group-ref parent 'level 0))
-                         0))
-    (test-group-set!
-     group
-     'skip-group?
-     (and (or (and parent (test-group-ref parent 'skip-group?))
-              (any (lambda (f) (f group)) (current-test-group-removers))
-              (and (null? (current-test-group-removers))
-                   (pair? (current-test-group-filters))))
-          (not (any (lambda (f) (f group)) (current-test-group-filters)))))
-    group))
+;; (name (prop . value) ...)
+(define (make-test-group name parent)
+  (let* ((g (list name))
+         (! (lambda (k v) (test-group-set! g k v))))
```

Comment:
This is fine, but in the future if you want a faster review of your changes please don't mix unrelated refactorings in with other changes :)

Reply:
Sorry about the noise. I’ll try to avoid this the next time.